File-Formats/DocIO/word-to-image.md

```diff
@@ -332,6 +332,319 @@ End Using
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-Image-conversion/Custom-image-resolution).
 
+## Fallback fonts
+
+During Word to Image conversions, if a glyph of the input text is unavailable in the specified font, the text will not be rendered properly. To address this, the Syncfusion Word (DocIO) library allows users to specify fallback fonts. When a glyph is missing, the library will use one of the fallback fonts to render the text correctly in the output image document.
```

Comment:
@LokeshBaskar - `output image document.` is not a proper phrase, please modify this as `output image.`

Reply:
Modified the content as output Image